spec/Modules.tex

```diff
@@ -182,14 +188,38 @@ \section{Nested Modules}
 \end{chapelexample}
 
 
-\section{Using Modules}
-\label{Using_Modules}
-\index{modules!using}
-
-A module can be used by code outside of that module.  This allows
-access to the top-level module symbols without the need for explicit
-naming~(\rsec{Explicit_Naming}).  Using modules is accomplished via
-the use statement as defined in~\rsec{The_Use_Statement}.
+\section{Access of Module Contents}
+\label{Access_Of_Module_Contents}
+\index{modules!access}
+
+A module can be used by code outside of that module depending on the
+visibility of the module itself~(\rsec{Visibility_Of_A_Module}).  This
+allows access to its visible contents~(\rsec{Visibility_Of_Symbols})
+without the need for explicit naming~(\rsec{Explicit_Naming}).  Using
+modules is accomplished via the use statement as defined
+in~\rsec{Using_Modules}.
+
```

Comment:
I would like the above paragraph to read like this: 1st sentence unchanged, then:

This
allows access to its visible contents~(\rsec{Visibility_Of_Symbols})
via explicit naming~(\rsec{Explicit_Naming})
or the use statement~(\rsec{Using_Modules}).